### PR TITLE
Server crash fixes

### PR DESCRIPTION
**Pull request by memosys**

### CODE REVIEW

File: docs/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN adduser --system --uid 1001 nextjs
 
 
 # Copy the bundled code from the build stage to the production image
-COPY --from=build --chown=nextjs:nodejs /app/.next/standalone ./
+# COPY --from=build --chown=nextjs:nodejs /app/.next/standalone ./
```
Comment (plor) [inline docs/Dockerfile:24]: I'm not a fan of commenting out code, if it isn't needed it should just be removed.

File: web-portal/backend/src/main.ts [hunk 2]
```diff
@@ -5,15 +5,11 @@ import { AppModule } from './app.module';
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   const { httpAdapter } = app.get(HttpAdapterHost);
-  app.useGlobalFilters(new PrismaClientExceptionFilter(httpAdapter));
+  // app.useGlobalFilters(new PrismaClientExceptionFilter(httpAdapter));
```
Comment (plor) [inline web-portal/backend/src/main.ts:8]: Another instance of commenting out code, not a fan.

**Review by plor — APPROVED** [review]
Mostly nitpicks, main question is if there is a way to catch Errors anywhere in the app and fallback to 500. Just worried that an error will slip through and crash in the future.

File: web-portal/backend/src/main.ts (hunk 2 above)
Comment (memosys) [reply web-portal/backend/src/main.ts:8]: understood ser, both are todo figure later!

**Pull request merged**